commands/motor/MotorControlAccelCap.java

```diff
@@ -83,15 +83,19 @@ protected void execute() {
 	}
 
 	@Override
-	protected boolean isFinished() {
+	public boolean isFinished() {
 		return false;
 	}
 
 	@Override
-	protected void end() {}
-
-	@Override
-	protected void interrupted() {
+	public void end(boolean interrupted) {
 		LogKitten.d("MotorControl interrupted");
```

Comment:
Again, I believe this code will make it log "MotorControl interrupted" whenever, the command ends, regardless of whether or not it was interrupted, and should probably be put inside an if statement.